feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.service.ts

```diff
@@ -48,6 +56,139 @@ export class ConfiguratorAttributeNumericInputFieldService {
       splitParts[1].length > numberDecimalPlaces
     );
   }
+
+  formatIntervalValue(
+    intervalValue: number,
+    decimalPlaces: number | undefined,
+    locale: string
+  ): string {
+    let formatted = formatNumber(
+      intervalValue,
+      locale,
+      '1.' + decimalPlaces + '-' + decimalPlaces
+    );
+    return formatted;
+  }
+
+  /**
+   * Parses the value names and returns the intervals.
+   *
+   * @param values values of the attribute
+   * @returns {ConfiguratorAttributeNumericInterval[]} parsed intervals
+   */
+  getIntervals(
+    values: Configurator.Value[] | undefined
+  ): ConfiguratorAttributeNumericInterval[] {
+    let intervals: ConfiguratorAttributeNumericInterval[] = [];
+    if (values && values.length > 0) {
+      intervals = [];
```

Comment:
This line is not needed. And then intervals can be defined with 'const'

Reply:
Done.